Automattic/wp-calypso
# Media: add a Guided Tour for media basics

This PR adds a tour that shows the basics of the Media Library to people. 

To test:
- start the tour by going to your site's media library and adding `?tour=mediaBasicsTour` to the URL (or just go here and select a site: https://calypso.live/media?branch=add/media-basics-tour&tour=mediaBasicsTour)
- check whether all steps display correctly (cf. GIF in comments below)
- check whether tour is understandable
- check whether steps look nice enough
- check whether the copy could be improved

TODO: 
- [x] guard against sites with no media files
- [x] send out request for another round of reviews
